src/Benchmarks/Data/BatchUpdateString.cs

```diff
@@ -10,48 +10,33 @@ namespace Benchmarks.Data
     internal class BatchUpdateString
     {
         private const int MaxBatch = 500;
+
         private static string[] _queries = new string[MaxBatch];
 
-        public static IList<BatchUpdateString> Strings { get; } =
-            Enumerable.Range(0, MaxBatch)
-                      .Select(i => new BatchUpdateString
-                      {
-                          Id = $"Id_{i}",
-                          Random = $"Random_{i}",
-                          BatchSize = i
-                      }).ToArray();
```

Comment:
Is there still value in keeping this, to string-interpolate `$"Id_{i}"` just once for each _i_, instead of once per query?